ui_framework/components/button/button.js

```diff
@@ -26,6 +26,7 @@ const buttonTypeToClassNameMap = {
   danger: 'kuiButton--danger',
   warning: 'kuiButton--warning',
   primary: 'kuiButton--primary',
+  secondary: 'kuiButton--secondary',
```

Comment:
I love that you're updating the React components too! On line 8 we need to make a change to match the change you've made here. Line 8 needs to be updated to be:

```jsx
const BUTTON_TYPES = [
  'basic',
  'hollow',
  'danger',
  'warning',
  'primary',
  'secondary',
];
```

This defines the allowed values, so now "secondary" will be accepted as a valid type.